nvidia_deeplearningexamples_waveglow.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: hub_detail
+background-class: hub-background
+body-class: hub
+title: NCF
```

Comment:
Title doesn't match model ;)

Reply:
sure. i already noticed it on netlify. corrected. thx